openstates/co/legislators.py

```diff
@@ -176,12 +174,13 @@ def scrape(self, chamber, session):
                 occupation=metainf['occupation'],
                 photo_url=metainf['photo_url'],
                 url=metainf['homepage'])
-            if "email" in metainf:
-                p['email'] = metainf['email']
-            if "number" in metainf:
-                p.add_office('capitol', 'Capitol Office',
-                             phone=metainf['number'],
-                             address='200 E. Colfax\nDenver, CO 80203'
+
+            phone = metainf['number'] if 'number' in metainf else None
+            email = metainf['email'] if 'email' in metainf else None
+            p.add_office('capitol', 'Capitol Office',
```

Comment:
nope, they just have the main address with no room/box on their legislator pages